tool/src/webapp/WEB-INF/requestContext.xml

```diff
@@ -248,6 +249,10 @@
 		<property name="lessonsAccess" ref="org.sakaiproject.lessonbuildertool.service.LessonsAccess"/>
 		<property name="lessonBuilderAccessService" ref="org.sakaiproject.lessonbuildertool.service.LessonBuilderAccessService" />
 	</bean>
+	<!-- Over-riding the CommonsMultipartResolver to have multiple files incoming under the same form input name  -->
+	<bean id="multipartResolver" class="org.sakaiproject.lessonbuildertool.resolver.MultiCommonsMultipartResolver">
+		<property name="maxUploadSize" value="100000"/>
```

Comment:
The maxUploadSize should come from the ServerConfigurationService: `content.upload.max` I think.